src/application.rs

```diff
@@ -182,9 +180,8 @@ pub trait ApplicationHandler {
     ///         }
     ///     });
     ///
-    ///     event_loop.run_app(&mut app)?;
+    ///     event_loop.run_app(MyApp { receiver })?;
```

Comment:
I feel like changes to this file are irrelevant for this PR and were not part of it